docs/tutorials/pytorch-mnist-tutorial.txt

```diff
@@ -156,20 +137,25 @@ The last two methods we need to define are ``build_training_data_loader`` and ``
 
 For more information on loading data in Determined, refer to the tutorial on :ref:`data-access`.
 
-Defining the Forward Pass
-"""""""""""""""""""""""""
+Defining the Training Function
+""""""""""""""""""""""""""""""
 
-The ``train_batch`` method computes the forward pass and calculates the loss for a single batch of data. Determined expects a dictionary with the calculated loss and other user-defined metrics and will automatically average all the metrics across batches. Since Determined handles the backpropagation call, we do not need to call ``loss.backwards()`` or ``optim.zero_grad()``.
+The ``train_batch`` method runs the forward passes on the losses and backward passes on the optimziers with a single batch of data. Determined expects a dictionary with the calculated loss and other user-defined metrics and will automatically average all the metrics across batches. Since ``step_optimizer`` handles the backpropagation call and zeroing out the gradients so there is no need to call ``optim.zero_grad()``.
```

Comment:
> runs the forward passes on the losses and backward passes on the optimziers with a single batch of data

Reword this somehow.  In reality you run the backward pass on the loss to calculate the gradients, and you step the optimizer.  I'm not sure what the right phrasing is for how to describe the forward pass.  "forward pass through the model", maybe?

> Determined expects a dictionary with the calculated loss and other user-defined metrics

The loss is no longer special, right?

> ``step_optimizer`` handles the backpropagation

Is this true?